schematic/store/synapse.py

```diff
@@ -606,7 +666,11 @@ def getProjectManifests(self, projectId: str) -> List[str]:
                         )
 
                     manifest_info = self.getDatasetManifest(datasetId,downloadFile=True)
-                    manifest_name = manifest_info["properties"]["name"]
+                    manifest_name = manifest_info["properties"].get("name", "")
+
+                    if not manifest_name:
```

Comment:
Great use of Python implicit booleanness!

Reply:
Thanks to your previous feedback.